tools/manifest/manifest.py

```diff
@@ -323,7 +323,7 @@ def update(self, tree):
 
         for source_file, update in tree:
             if not update:
-                assert isinstance(source_file, (bytes, str))
+                assert isinstance(source_file, (bytes, str, unicode))
```

Comment:
Let's just make this `(binary_type, text_type)`, importing both from six. Ignore the nonsense I wrote before.

Reply:
Done.